tsuru/client/auth.go

```diff
@@ -478,41 +491,49 @@ Tags: {{.Tags}}
 
 type ChangePassword struct{}
 
-func (c *ChangePassword) Run(context *cmd.Context, client *cmd.Client) error {
-	u, err := cmd.GetURL("/users/password")
-	if err != nil {
-		return err
+func (c *ChangePassword) ChangePassword(old, new, confirm string) tsuru.ChangePasswordData {
+	changePass := tsuru.ChangePasswordData{
+		Old:     old,
+		New:     new,
+		Confirm: confirm,
 	}
-	fmt.Fprint(context.Stdout, "Current password: ")
-	old, err := cmd.PasswordFromReader(context.Stdin)
+	return changePass
+}
+func (c *ChangePassword) Run(ctx *cmd.Context, client *cmd.Client) error {
+	ctx.RawOutput()
+
+	fmt.Fprint(ctx.Stdout, "Current password: ")
+	old, err := cmd.PasswordFromReader(ctx.Stdin)
 	if err != nil {
 		return err
 	}
-	fmt.Fprint(context.Stdout, "\nNew password: ")
-	new, err := cmd.PasswordFromReader(context.Stdin)
+	fmt.Fprint(ctx.Stdout, "\nNew password: ")
+	new, err := cmd.PasswordFromReader(ctx.Stdin)
 	if err != nil {
 		return err
 	}
-	fmt.Fprint(context.Stdout, "\nConfirm: ")
-	confirm, err := cmd.PasswordFromReader(context.Stdin)
+	fmt.Fprint(ctx.Stdout, "\nConfirm: ")
+	confirm, err := cmd.PasswordFromReader(ctx.Stdin)
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(context.Stdout)
+	fmt.Fprintln(ctx.Stdout)
 	v := url.Values{}
 	v.Set("old", old)
 	v.Set("new", new)
 	v.Set("confirm", confirm)
-	request, err := http.NewRequest("PUT", u, strings.NewReader(v.Encode()))
+	apiClient, err := tsuruClient.ClientFromEnvironment(&tsuru.Configuration{
+		HTTPClient: client.HTTPClient,
+	})
 	if err != nil {
 		return err
 	}
-	request.Header.Set("Content-Type", "application/x-www-form-urlencoded")
-	_, err = client.Do(request)
+	changePassword := c.ChangePassword(old, new, confirm)
```

Comment:
this variable name does not make any sense... changePassword->changedPassword or changePassword->newPassword